src/styles/LeftSideBar.css

```diff
@@ -81,9 +81,15 @@ input:checked + .slider:before {
 
 .panel {
     padding: 0 60px;
-    display: none;
+    cursor: pointer;
+    -webkit-transition: 0.4s;
```

Comment:
We might need some tool to handle cross browser compatibility, so doing this won't be necessary anymore.